tests/value/test_semivalues.py

```diff
@@ -48,12 +48,13 @@ def test_shapley(
     parallel_config: ParallelConfig,
 ):
     u, exact_values = analytic_shapley
-    criterion = AbsoluteStandardError(0.02, 1.0) | MaxUpdates(2 ** (num_samples * 2))
+    criterion = AbsoluteStandardError(1e-4) | MaxUpdates(1000)
     values = compute_generic_semivalues(
         sampler(u.data.indices),
         u,
         coefficient,
         criterion,
+        skip_converged=True,
```

Comment:
Case `skip_converge=False` missing. Maybe better to include a specific test case for skip converged.

Reply:
Hmmm... maybe. I didn't want to add more tests. In a sense it contains skip=True as a special case where no indices converge before all do (yes, that's a stretch)